llvm/lib/Target/RISCV/RISCVProcessors.td

```diff
@@ -381,3 +381,20 @@ def XIANGSHAN_NANHU : RISCVProcessorModel<"xiangshan-nanhu",
                                             TuneZExtHFusion,
                                             TuneZExtWFusion,
                                             TuneShiftedZExtWFusion]>;
+
+def SPACEMIT_X60 : RISCVProcessorModel<"spacemit-x60",
+                                       NoSchedModel,
+                                       !listconcat(RVA22S64Features,
+                                       [FeatureStdExtV,
+                                        FeatureStdExtSscofpmf,
+                                        FeatureStdExtSstc,
+                                        FeatureStdExtSvnapot,
+                                        FeatureStdExtZbc,
+                                        FeatureStdExtZbkc,
+                                        FeatureStdExtZfh,
+                                        FeatureStdExtZicond,
+                                        FeatureStdExtZmmul,
```

Comment:
Please don't list Zmmul separate from M.

Reply:
Generated `"-target-feature" "-zmmul"` in `clang/test/Driver/riscv-cpus.c`.
It seems that M imply Zmmul is not supported yet.
https://github.com/llvm/llvm-project/pull/95070

Reply:
> Generated "-target-feature" "-zmmul" in clang/test/Driver/riscv-cpus.c.
It seems that M imply Zmmul is not supported yet.
https://github.com/llvm/llvm-project/pull/95070

It's not, but what is the benefit to the user to have Zmmul and M both listed?